src/main/java/sh/platform/config/Route.java

```diff
@@ -0,0 +1,65 @@
+package sh.platform.config;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Optional;
+
+public class Route {
+
+    private String id;
+
+    private String upstream;
+
+    @SerializedName("original_url")
+    private String originalUrl;
+
+    @SerializedName("restrict_robots")
+    private boolean restrictRobots;
+
+    private boolean primary;
+
+    private String type;
+
+    private String to;
+
+    public Optional<String> getId() {
+        return Optional.ofNullable(id);
+    }
+
+    public Optional<String> getUpstream() {
+        return Optional.ofNullable(upstream);
+    }
+
+    public String getOriginalUrl() {
+        return originalUrl;
+    }
+
+    public boolean isPrimary() {
+        return primary;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public boolean isRestrictRobots() {
+        return restrictRobots;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Is this string useful for something other than debugging?  I'm not sure what the value is here.

Reply:
Yeap, to debugging.
this `toString` is better than show the memory address

Reply:
Is it standard Java convention to include a debugging toString like that?

Reply:
Yeap,

Item 10 - Always override toString
From Effective Java  by Joshua Bloch
http://thefinestartist.com/effective-java/10

Reply:
I usually don't put in all, but, I put on the key objects such as Route class.